DataFormats/Detectors/TPC/src/ClusterNativeHelper.cxx

```diff
@@ -14,12 +14,20 @@
 /// @author Matthias Richter
 
 #include "DataFormatsTPC/ClusterNativeHelper.h"
+#include "DataFormatsTPC/TPCSectorHeader.h"
+#include "DataFormatsTPC/Digit.h"
+#include "Framework/ProcessingContext.h"
+#include "Framework/DataRefUtils.h"
+#include "Framework/InputRecordWalker.h"
```

Comment:
I think the DataFormat definitions and their primitive helpers should be free from Framework stuff. It looks like we have this dependency already because nothing else was necessary to be added in CMakeLists. But we should not add more then we need and some point we want to clean up the dependencies.

Reply:
I agree and was actually astonished that the framework dependency was already there, but since it was there I just used it. Fine with me to move all the framework stuff somewhere else.
Personally, I'd create a TPCWorkflowHelper library containing the stuff I added here and also the Completion Policy helper you added.